docker-compose.dev.yml

```diff
@@ -21,6 +21,8 @@ services:
     tty: true
     volumes:
       - .:/usr/src/app
+    environment:
+      - LISTEN=0.0.0.0
```

Comment:
If the container is solely for running unit tests, then these lines could be removed.